### ISSUE TITLE
[MAS2.1.1] Unable to close any tab using keyboard.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-Emulator#731]
`**Actual Result:**
All tab buttons like 'Welcome', 'Settings' and 'Bot1 <Name of any bot which is currently running>' are not accessible using keyboard.

**Expected Result:**
All tab buttons like 'Welcome', 'Settings' and 'Bot1 <Name of any bot which is currently running>' should be accessible using keyboard.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Click on open bot and choose the .bot file that is available in the bot folder.
3. Go to Settings button present in side pane and select it.
4. Notice all these tabs present at the top.
5. Try to navigate and select all these tab buttons using keyboard.

**User Impact:**
Keyboard dependent users won't be able to use the controls which are not accessible using keyboard in any way.

**Test Environment:**
OS: Windows 10
OS Build: 17134.167
OS Version: 1803
Browser: Edge

**Impacted MAS Link:**
[MAS 2.1.1 – Keyboard (36)](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2})

**Attachment:**
[MAS2.4.7_BotFrameworkEmulator_TabButtons.zip](https://github.com/Microsoft/BotFramework-Emulator/files/2263010/MAS2.4.7_BotFrameworkEmulator_TabButtons.zip)

**Comment by corinagum** [comment]
Fixed
